### ISSUE TITLE
Documentation request: removing multiple axioms types

### ISSUE THREAD

**Issue by matentzn** [ontodev/robot#634]
I have been using remove successfully for a while like:

```
$(ROBOT) remove -i $< --axioms DisjointClasses --output $@
```

However, based on the statement `More than one type can be provided and these will be processed in order` in the [docs](http://robot.obolibrary.org/remove#axioms) I assumed the following was possible:

```
$(ROBOT) remove -i $< --axioms "DisjointClasses DisjointUnion" --output $@
```

However, in the second example, nothing gets removed, not even the `DisjointClasses` (the first example works normally). Can you provide an example in the documentation on how to remove multiple axioms types? Thanks!

**Comment by jamesaoverton** [comment]
My guess is that this is a bug with multiple axioms types. @beckyjackson will check.

Note that there's a `--axioms disjoint` for removing all kinds of disjoints, but that might be more than you want here.

**Comment by beckyjackson** [comment]
Thanks for catching this. Yes, this is a bug. It looks like in a list of multiple axiom types, only the *last* axiom type is actually used. In this case, `DisjointUnion`.

I've made a fix for this in #644

If you're able to, please test it out with the JAR here: https://build.obolibrary.io/job/ontodev/job/robot/job/634-fix/lastSuccessfulBuild/artifact/bin/robot.jar

Thank you!

**Comment by matentzn** [comment]
This is amazing thanks! and congrats on the new ROBOT release!

I just reviewed the pull request! 

Given this: 

```
filteredAxioms.addAll(
            filterAxiomsByAxiomType(inputAxioms, objects, axiomTypes, partial, namedOnly));
```

I trust that this works :P Thanks a lot! And thanks for adding the docs as well, this is really helpful!
